### PR TITLE
[EN] Game Modifiers

### PR DESCRIPTION
**Pull request by RaikireHiuduo**
**Ready for review/merge**

Taking over #486

~~Icon rely on #435 .~~ Updated

~~Planning on some rewrites but mostly on the structure formatting.~~ Full rewrite.

File changes (en only):-
- Game_Modifers
  - Non-English pages are purged without blinking an eye in 10 seconds.
- Index (Add VS)
- Visual_Settings (added)
- ~~Score (fixed ScoreV2 link)~~ #527 

Format (modified from @MegaApplePi idea by PM):-

```Markdown
### modName

{icon}

> {If specific to mode, **{modeName} only**}

> Score Multiplier: {score}

> Shortcut Key: {shortcutKey}

> Description: {generalDescriptionUsedInAllModes}

{generalExplanation}

{image (when needed)}

---
(duplicate this as needed)

{image (when needed)}

> Description: {descriptionSpecificToMod}

{osu!/osu!taiko/osu!catch/osu!mania Explanation}


### nextModName
```

### CODE REVIEW

**Comment by RaikireHiuduo** [comment]
Yes. This PR is not in mergeable state yet. I hit a roadblock in writing and have not added images yet.

---

Key problems:-

- {{{{WOT}}}} = I have no idea what it meant or I literally have no idea what to do and I need help. (Use `Ctrl`+`F` on en.md, specifically on DT)
- HR <--- Amazingly, CS: 9.1 (from 7) is possible (13.64 circle/fruit radius).
- DT <--- Especially this one. Approach time and scoring timing changes. Also, no clue how to write it.

Minor problems:-

- Target Practice description and the shortcut key in the Game Modifier screen to enable the mod.
- Since the whole page is modified, should I delete all the other language files?

File: wiki/Game_Modifiers/en.md [hunk 1]
```diff
@@ -1,306 +1,531 @@
...
+It also allows three (3) "lives"; when the [life bar][lifebar wikilink] is exhausted, the game pauses and it refills up to about 80% of the health bar for two times. This restart will stop the song temporary to fill up the HP bar and the sound hint is the `ready` sound (depends on skin used). In addition, replays will not show the restart or restored HP so if a replay is watched where the player had to use a "life", it will appear as though the player continues to play with no health. It does **not** work in [Tag Coop / Tag Team Vs Team Mode in Multi](/wiki/Multi/#tag-coop---tag-team-vs)
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: Suggestion:
```
It also gives the player two additional "lives"; if the [life bar][lifebar wikilink] is exhausted, the game will pause and refills the health bar to about 80%. During this time, a sound will `ready` sound (if skinned) will play to hint that the game will continue.

Interestingly, if someone were to watch a replay with the player using an extra "life" is not shown; rather, it is skipped while showing that the player had ran out of health.

In [multi](/wiki/multi) mode, the effect of extra lives will not work in [Tag Coop / Tag Team Vs](/wiki/multi/#tag-coop-tag-team-vs) modes.
```

File: wiki/Game_Modifiers/en.md [hunk 2]
```diff
@@ -1,306 +1,531 @@
...
 
-However, some (if not most) argue that Easy Mod fails to make maps easier to play. They claim that circles become comically large, and AR becomes ridiculously slower. The mod has the potential of making the play area much more cluttered and harder to read, especially on [insane](/wiki/Difficulties/osu!/Insane) or approved beatmaps. These points are elaborated upon in [Let's talk Easy Mod](https://osu.ppy.sh/forum/t/56606) feature request to improve the mod, by [Lybydose](https://osu.ppy.sh/u/64501).
+However, some (if not most) argue that Easy Mod fails to make beatmaps easier to play. The claim is that circles become comically large, and AR becomes ridiculously slower. The mod has the potential of making the play area much more cluttered and harder to read, especially on [insane](/wiki/Difficulties/osu!/Insane) or approved beatmaps. These points are elaborated upon in [Let's talk Easy Mod](https://osu.ppy.sh/forum/t/56606) feature request to improve the mod, by [Lybydose](https://osu.ppy.sh/u/64501).
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: remove this; another comment below will say where to put it

File: wiki/Game_Modifiers/en.md [hunk 3]
```diff
@@ -1,306 +1,531 @@
...
+---
 
-In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the effects are the same as in osu! mode - all fruits fall slower (same as reducing AR in osu! mode), the camera is zoomed in (increased circle size in osu! mode) and 3 lives are granted. Despite only using one axis, however, the fruits' larger size and slower falling rate causes them to clump together, creating the same alleged problems as in normal osu!
+> Description: Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: remove `Description: `?

File: wiki/Game_Modifiers/en.md [hunk 4]
```diff
@@ -1,306 +1,531 @@
...
 These mods make the game easier at the cost of decreasing the score obtained.
 
 ### Easy
 
-![Easy mod icon](/wiki/shared/Easy.png "Easy mod icon")
+![Easy (EZ) mod icon][EZ]
 
-**Score multiplier**: 0.5
+> Score multiplier: 0.50
+> Shortcut key: `Q`
```
Comment (RaikireHiuduo) [inline wiki/Game_Modifiers/en.md]: Gonna keep as default.

ScoreV2 and Random do not have Shortcut key.

Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: then add an extra space above; on GitHub, it puts them both together

File: wiki/Game_Modifiers/en.md [hunk 5]
```diff
@@ -1,306 +1,531 @@
...
 
-**Description**: Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.
+Easy mod halves all difficulty settings of the beatmap selected at a cost of decreasing the score modifier by half.
```
Comment (RaikireHiuduo) [inline wiki/Game_Modifiers/en.md]: Simplified to

Easy mod halves all difficulty settings from the beatmap selected.

File: wiki/Game_Modifiers/en.md (hunk 2 above)
Comment (RaikireHiuduo) [inline wiki/Game_Modifiers/en.md]: Movin' it to da Trivia.

File: wiki/Game_Modifiers/en.md [hunk 6]
```diff
@@ -1,306 +1,531 @@
...
 
 Game modifiers can be accessed by pressing `F1` at song select, or by clicking the `Mods` button near the lower left of the screen.
 
-Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. *osu!* used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x.
+![Song Selection's Toolbox][Toolbox]
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: red arrows don't look great on images :/ ... maybe shrink down the cursor?
screenshot167

**Comment by RaikireHiuduo** [comment]
Status notes:-

On-topic:-

1. Check Double Time. The whole explanation for Double Time. Also, Half Time.
2. The images of EZ. It feels odd to me and I am thinking the current image description is at wrong places (no idea how the old image's pattern is supposed to look like; I do not know if `old-wiki` keep old image history).
3. Target Practice: Description and shortcut key used.

Off-topic:-

1. I need the `Beatmap_Editor/Song_Setup` page to be updated since EZ/HR and HT/DT mess with the difficulty settings and there is one more specific values are not documented (like the CS affecting circle/fruit radius). Also, pressing `Shift` will fine-tune the increment value to _0.1_ increment rather than 1 increment (meaning, there is a lot of values to document without the formula known unless we only account main values only).

---

CS/Mod | EZ | HR
---|---|---|---
0 | 0   | 0
1 | 0.5 | 1.3
2 | 1   | 2.6
3 | 1.5 | 3.9
4 | 2   | 5.2
5 | 2.5 | 6.5
6 | 3   | 7.8
7 | 3.5 | 9.1


CS | Circle/Fruit Radius
---|---
0   | 54.42
0.5 | 52.18
1   | 49.94
1.3 | 48.6
1.5 | 47.7
2   | 45.46
2.5 |43.22
2.6 | 42.77
3   | 40.98
3.5 | 38.74
3.9 | 36.94
4   | 36.49
5   | 32.01
5.2 | 31.12
6   | 27.53
6.5 | 25.29
7   | 23.05
7.8 | 19.46
9.1 | 13.64

File: wiki/Game_Modifiers/en.md [hunk 7]
```diff
@@ -1,306 +1,608 @@
...
+
+> Score multiplier: 0.50
+
+> Shortcut key: `Q`
+
+Easy mod halves all difficulty settings of the beatmap selected.
+
+The Easy mod also grants the player two (2) additional "lives"; if the [life bar][lifebar wikilink] is exhausted completely, the play will pause and refills the life bar slowly to about 80% with no sound in-between at a cost of one "life" of the play. The refill is signaled by the ready sound (`readys.wav`) instead of the typical fail sound (`failsound.wav`) for failing the beatmap with an empty life bar. The beatmap will resume once the refill is successful signaled by playing the go sound (`gos.wav`). The play will fail instead if there is no "lives" remain when the life bar has been exhausted completely again.
+
+Interestingly, when watching a replay where extra "life" was used, the replay will skips over the health refill animation and allows the replay to continue at no health instead while allowing health regain from hit objects.
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: draft of replacement:
`Interestingly, when watching a replay where an extra "life" was used, the replay will skip over the health refill animation and keep on going without any health instead while allowing health regain from hit objects.`

File: wiki/Game_Modifiers/en.md [hunk 8]
```diff
@@ -1,306 +1,608 @@
...
+
+In [Multi](/wiki/multi "Multi") mode, the effect of extra lives will not work in [Tag Coop / Tag Team Vs](/wiki/multi/#tag-coop-tag-team-vs "Tag Coop / Tag Team Vs") team mode.
+
+---
+
+> Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.
 
-**Score multiplier**: 0.5
+In [osu!standard][osu! wikilink] mode, the Easy mod decreases [circle size][CS wikilink] (CS; lower values means larger circles), [approach rate][AR wikilink] (AR), [overall difficulty][OD wikilink] (OD) as well as [HP drain][HP wikilink] (HP); each of these by a huge amount (half of the current amount set).
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: don't explain what CS is; that's what the Song Selection article is supposed to do :/

Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: remove `mode` after `osu!standard` they are already a game mode; so it seems overkill to state `mode` again
(suppressing future instances)

**Pull request merged**